extensions/dicom-seg/src/init.js

```diff
@@ -0,0 +1,20 @@
+import csTools from 'cornerstone-tools';
+
+/**
+ *
+ * @param {object} configuration
+ * @param {Object|Array} configuration.csToolsConfig
+ */
+export default function init({ servicesManager, configuration = {} }) {
+  const { BrushTool, SphericalBrushTool, CorrectionScissorsTool } = csTools;
+  const tools = [BrushTool, SphericalBrushTool, CorrectionScissorsTool];
+
+  tools.forEach(tool => csTools.addTool(tool));
+
+  csTools.addTool(BrushTool, {
+    name: 'BrushEraser',
+    configuration: {
+      alwaysEraseOnClick: true,
+    },
+  });
+}
```

Comment:
@JamesAPetts this works well when we control these extensions, but I still worry about collisions here across many extensions.

Reply:
Right now this is a problem for any extension that adds tools. I don't think its a problem at this time due to the limited scope of extensions. This won't be an issue in the future if extensions are basically pools of components, as you'll pull in the implementations you want in your mode.

At the current time when everything is self registering, I don't think this is too much of a big deal.